data.go

```diff
@@ -162,6 +176,10 @@ func ParseOptions(str string) Options {
 		}
 	}
 
+	if options.Quality == 0 {
+		options.Quality = 95
```

Comment:
actually, it looks like you are already checking for the zero value in `transformImage`, so this check here shouldn't be needed at all I don't think.